.travis.yml

```diff
@@ -1,12 +1,26 @@
 language: c
 sudo: false
 dist: trusty
-compiler:
-  - clang
-  - gcc
-os:
-  - linux
-  - osx
+# We can't use strict flags in CFLAGS as a general environment
+# variable, because that messes up ./configure.
```

Comment:
err, right.  We need this for scrypt and tarsnap (eventually), but not other projects.